src/components/SiteNav/SiteNav.module.css

```diff
@@ -61,4 +67,12 @@
 
 /* Desktop styling */
 @media (min-width: 786px) {
```

Comment:
This value should possibly be changed, I think the desktop version could be used on some smaller viewports. But will probably make more sense to tweak if and when we do the proper burger menu

Reply:
The Figma gives us some suggested 'break-points' of >1024, >768 and > 320

I just went with an arbitrary choice:

![Screenshot from 2024-05-21 11-55-16](https://github.com/technative-academy/PetSynth/assets/143351775/e3d9ca58-5867-43de-be30-cb302a48713c)

Reply:
though I do not another Conrad classic of inverting the digits 768 v 786

Reply:
Ah, I missed that. I think I'll ask Pete since my approach so far and previously has been to change it until it seems to break at a logical point when resizing